CONTRIBUTING.md

```diff
@@ -1,3 +1,92 @@
+# Contributing Guide for the 'Run Salesforce Code Analyzer' GitHub Action
+> Currently, we aren't accepting external contributions. To know when we open for contributions, periodically check this guide.
 
+To contribute to the Run Code Analyzer GitHub Action, read this guide to understand its operational governance model. This guide also provides contribution requirements and recommendations. Thanks for your interest and contributions.
+
+# Governance Model
+
+## Salesforce Sponsored
+
+The intent and goal of open sourcing this project is to increase the contributor and user base. Note that Salesforce employees have `admin` rights and are the final arbiters which contributions are accepted.
+
+# Issues, requests & ideas
+
+To submit issues, enhancement requests, and discuss ideas, create an issue on the [Issues page](https://github.com/forcedotcom/run-code-analyzer/issues).
+
+### Bug Reports and Fixes
+-  If you find a bug, search for it in the [Issues](https://github.com/forcedotcom/run-code-analyzer/issues), and if it isn't already tracked,
+   [create a new issue](https://github.com/forcedotcom/run-code-analyzer/issues/new). Fill out the "Bug Report" section of the issue template. Even if an issue is closed, feel free to comment and add details, it will still
+   be reviewed.
+-  Issues that have been confirmed as a bug will be labelled `bug`.
+-  If you'd like to submit a fix for a bug, [create a Pull Request](#creating_a_pull_request) and mention the issue number.
+-  In your bug fix pull request, include tests that isolate the bug and verify that it's fixed.
+
+### New Features
+-  To add new functionality to Run Code Analyzer, describe the problem you want to solve (or the enhancement you identifed) in a [new issue](https://github.com/forcedotcom/run-code-analyzer/issues/new).
+-  Issues that are identified as a feature request will be labelled `enhancement`.
+-  Before writing the code for your newly requested feature, first wait for feedback from the project maintainers. In some cases, requested `enhancements` don't align well with the current project objectives.
+
+### Tests, Documentation, Miscellaneous
+-  We welcome your contributes to:
+   - imporiving tests
+   - clarifying documentation
+   - requesting alternative implementations
+-  If its a trivial change, go ahead and [create a pull request](#creating_a_pull_request) with the changes you have in mind.
+-  If your request is larger in size, [open an issue](https://github.com/forcedotcom/run-code-analyzer/issues/new) to get feedback on your idea.
+
+# Contribution Checklist
+
+- [x] Clean, simple, well styled code
+- [x] Atomic commits and descriptive messages. Mention related issues by issue number
+- [x] Comments
+    - Module-level & function-level comments
+    - Comments on complex blocks of code or algorithms (include references to sources)
+- [x] Tests
+    - Complete, passing test suite test suite (if provided)
+    - Increase code coverage, not versa
```

Comment:
What do you mean by "versa"?

Increase code coverage, not decrease?

Reply:
Yeah maybe that's what the oss folks meant. But how about: "Maintain or increase code coverage percentages"